prelude/toolchains/msvc/vswhere.py

```diff
@@ -216,6 +286,8 @@ def main():
         lib_exe = find_in_path("lib.exe")
         ml64_exe = find_in_path("ml64.exe")
         link_exe = find_in_path("link.exe")
```

Comment:
With const list this can be simplified.
```suggestion
        cl_exe, lib_exe, ml64_exe, link_exe = (find_in_path(exe) for exe in EXE_NAMES)
```